### PR TITLE
Add Fio banka

### PR DESCRIPTION
**Pull request by radeksimko**
The bank does implement SMS-based verification, but it's only for authorization of transactions, not for login.

https://www.fio.cz/bank-services/internetbanking/security

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -433,6 +433,13 @@ websites:
       img: fifththird.png
       tfa: No
 
+    - name: Fio.cz
+      url: https://www.fio.cz
+      twitter: Fio_banka
+      facebook: Fiobanka
+      img: fio-cz.png
+      tfa: No
+
```
Comment (kmpoppe) [inline _data/banking.yml]: Please add `lang`-field.

Comment (kmpoppe) [reply _data/banking.yml]:
```suggestion
      lang: cz
      
```

**Review by kmpoppe — DISMISSED** [review]

**Comment by jtagcat** [comment]
I suggest you to reduce the png size with this image.
fio-cz

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Change the image to the one provided in https://github.com/2factorauth/twofactorauth/pull/3809#issuecomment-483598082

**Comment by radeksimko** [comment]
@Carlgo11 @kmpoppe PTAL, I implemented both suggestions.

**Review by kmpoppe — DISMISSED** [review]

File: _data/banking.yml [hunk 2]
```diff
@@ -433,6 +433,14 @@ websites:
       img: fifththird.png
       tfa: No
 
+    - name: Fio.cz
```
Comment (Carlgo11) [inline _data/banking.yml]: On both Twitter and Facebook they refer to themselves as "Fio banka" so I suggest you change the name tag to reflect that.
```suggestion
    - name: Fio banka
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Review by kmpoppe — APPROVED** [review]

**Pull request merged**